Support using libraries which have been moved into ports, not bundled
-lSDL2_image is in CMAKE_EXE_LINKER_FLAGS (see port Makefile).

Index: CMake/Dependencies.cmake
--- CMake/Dependencies.cmake.orig
+++ CMake/Dependencies.cmake
@@ -85,40 +85,6 @@ else()
   endif()
 endif()
 
-macro(_find_SDL_image QUIET_OR_REQUIRED)
-  # vcpkg uses sdl2-image as the package name
-  find_package(sdl2-image QUIET)
-  set(SDL_image_FOUND ${sdl2-image_FOUND})
-
-  if(NOT SDL_image_FOUND)
-    # Fall back on PkgConfig via FindSDL2_image.cmake
-    find_package(SDL2_image ${QUIET_OR_REQUIRED})
-    set(SDL_image_FOUND ${SDL2_image_FOUND})
-  endif()
-endmacro()
-
-if(NOT USE_SDL1)
-  if(EMSCRIPTEN)
-    emscripten_system_library("SDL_image" SDL2::SDL2_image USE_SDL_IMAGE=2 "SDL2_IMAGE_FORMATS='[\"png\"]'")
-  else()
-    if(NOT DEFINED DEVILUTIONX_SYSTEM_SDL_IMAGE)
-      _find_SDL_image(QUIET)
-      if(SDL_image_FOUND)
-        message("-- Found SDL_image")
-      else()
-        message("-- Suitable system SDL_image package not found, will use SDL_image from source")
-        set(DEVILUTIONX_SYSTEM_SDL_IMAGE OFF)
-      endif()
-    endif()
-    dependency_options("SDL_image" DEVILUTIONX_SYSTEM_SDL_IMAGE ON DEVILUTIONX_STATIC_SDL_IMAGE)
-    if(DEVILUTIONX_SYSTEM_SDL_IMAGE)
-      _find_SDL_image(REQUIRED)
-    else()
-      add_subdirectory(3rdParty/SDL_image)
-    endif()
-  endif()
-endif()
-
 if(NOT DEFINED DEVILUTIONX_SYSTEM_LIBFMT)
   find_package(fmt 8.0.0 QUIET)
   if(fmt_FOUND)
@@ -163,7 +129,7 @@ if(PACKET_ENCRYPTION)
   endif()
 endif()
 
-add_subdirectory(3rdParty/libsmackerdec)
+#add_subdirectory(3rdParty/libsmackerdec)
 
 if(WIN32 AND NOT UWP_LIB)
   add_subdirectory(3rdParty/find_steam_game)
@@ -185,20 +151,12 @@ else()
   add_subdirectory(3rdParty/simpleini)
 endif()
 
-if(SUPPORTS_MPQ)
-  add_subdirectory(3rdParty/libmpq)
-endif()
-
 add_subdirectory(3rdParty/tl)
 
 add_subdirectory(3rdParty/hoehrmann_utf8)
 
 if(SUPPORTS_MPQ OR NOT NONET)
   add_subdirectory(3rdParty/PKWare)
-endif()
-
-if(NOT NONET AND NOT DISABLE_TCP)
-  add_subdirectory(3rdParty/asio)
 endif()
 
 if(NOT NONET AND NOT DISABLE_ZERO_TIER)
